tests/integration/utils.py

```diff
@@ -275,3 +276,47 @@ def assert_content_manifest(client, request_id, image_contents):
     response_data = content_manifest_response.data
     assert_content_manifest_schema(response_data)
     assert image_contents == content_manifest_response.data["image_contents"]
+
+
+def package_manager_common_checks(test_env, env_data, tmpdir):
```

Comment:
Can we split this into more functions instead?

Reply:
I also like the approach when we don't have big function for test cases and have small functions that will these test use instead. It's more variable.